dotcom-rendering/cypress/integration/parallel-1/article.elements.spec.js

```diff
@@ -106,6 +106,13 @@ describe('Elements', function () {
 				'Article?url=https://www.theguardian.com/sport/blog/2015/dec/02/the-joy-of-six-sports-radio-documentaries',
 			);
 
+			cy.scrollTo(0, 4500);
```

Comment:
We're scrolling to a specific pixel position - will this always yield the next expected action (which I'm assuming scrolls something into view that then gets hydrated)?

Reply:
This is the expected behaviour, yes. Once our element is in view it becomes hydrated and we can continue the test verifying it does what we expect. The risk is content on this article moves around but that's unlikely (I hope!)